aten/src/ATen/core/ATenCoreTest.cpp

```diff
@@ -1,9 +1,11 @@
 #include <ATen/core/ATenCoreTest.h>
+#include <ATen/core/Tensor.h>
 
 namespace at {
 
 static int CoreTestGlobal = 0;
 int CoreTest() {
+  Tensor x;
```

Comment:
Why?

Reply:
Otherwise Tensor is never actually tested in a "core only" context (it's not hooked up to anything on mobile).